### DISCUSSION TITLE
GC Affiliation

### DISCUSSION THREAD

**Discussion by rburrowes** [Adventech/sabbath-school-android#194] [General]
Hello! 
It seems that I am the 1st to start a discussion here. I am Raymond, a member of the SDA church and a software QA in my occupation. I am really grateful and impressed with the app. It has been a blessing to me, and I have been able to bless others with the word of God via this platform. God bless you all 🙏🏾
I am just here to ask a quick question for clarification.🙊

Is the app now "affiliated" with the General Conference or now supported by General Conference?

It does not seem clear what is the current relationship and involvement is since the app has been rebranded.
On the website it states, "... we are not affiliated with the General Conference."
![Screenshot_20210513_055433_com android chrome](https://user-images.githubusercontent.com/26167747/118075922-544f5280-b3b1-11eb-9d5e-5b0fba036cc2.jpg)
But the in app message is that it is now "The official General Conference Sabbath School and Personal Ministries app.", 
![Screenshot_20210513_055648_com android vending](https://user-images.githubusercontent.com/26167747/118076045-9bd5de80-b3b1-11eb-81cb-ec27a09943c0.jpg)

Is that the technology for the app is provided by Adventech and the content provided by the General Conference?
Or is it that the technology and content is all provided by Adventech, which has been officially approved by the General Conference?

Thanks in advance for any responses and/discussion on this subject.
May God continue to bless your efforts to spread the gospel message.🙏🏾

**Comment by VitalikL** [comment]
Hi @rburrowes. We are happy to announce our partnership with the GC. As of recently, this now is the official app for Sabbath School and Personal Ministries for the General Conference. I have updated the footer copy on our website to reflect that.
